editor/code_editor.cpp

```diff
@@ -419,6 +419,26 @@ void FindReplaceBar::_update_matches_label() {
 	}
 }
 
+void FindReplaceBar::_temp_unfold_line(int &p_line) {
```

Comment:
Where is `p_line` modified?

Reply:
Oh, nevermind, I didn't pay enough attention to this remark in the last review, sorry. I wrote this because I was slightly worse with C++ back when I made this PR.